phpcs.xml.dist

```diff
@@ -45,6 +45,15 @@
 		</properties>
 	</rule>
 
+	<!-- Verify that no WP functions are used which are deprecated or have been removed.
+		 The minimum version set here should be in line with the minimum WP version
+		 as set in the "Requires at least" tag in the readme.txt file. -->
+	<rule ref="WordPress.WP.DeprecatedFunctions">
+		<properties>
+			<property name="minimum_supported_version" value="4.0" />
```

Comment:
Yes, to quote myself:
_The minimum version set here should be in line with the minimum WP version as set in the "Requires at least" tag in the readme.txt file._

And the `readme.txt` file says `4.0` at the moment.
https://github.com/Automattic/_s/blob/master/readme.txt#L6